.github/actions/smoke-tests/action.yml

```diff
@@ -0,0 +1,49 @@
+name: Smoke test
+description: "Execute smoke tests"
+
+inputs:
+  submit-url:
+    description: 'Submit layer url'
+    required: true
+  direct-submit-url:
+    description: 'Direct submission URL to Backtrace instance'
+    required: true
+  sauce-username:
+    description: 'Sauce labs user name'
+    required: true
+  sauce-access-key:
+    description: 'Sauce labs secret'
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - uses: dorny/paths-filter@v3
+      id: filter
+      with:
+          filters: |
+                browser:
+                    - 'packages/browser/**'
+                    - 'packages/sdk-core/**'
+                    - 'packages/sdk-core/**'
+                node:
+                    - 'packages/sdk-core/**'
+                    - 'packages/electron/**'
+                    - 'packages/nestjs/**'
```

Comment:
I think there's no point in rerunning node smoketest if electron/nestjs changes.